README.md

```diff
@@ -1,7 +1,7 @@
-# Salesforce Code Analyzer Extension for Visual Studio Code (Beta)
+# Salesforce Code Analyzer Extension for Visual Studio Code
 Scan your code against multiple rule engines to produce lists of violations and improve your code.
 
-The Salesforce Code Analyzer Extension (beta) enables Visual Studio (VS) Code to use Salesforce Code Analyzer to interact with your code.
+The Salesforce Code Analyzer Extension enables Visual Studio (VS) Code to use Salesforce Code Analyzer to interact with your code.
 
 # Documentation
 For documentation, visit the [Salesforce Code Analyzer VS Code Extension](https://forcedotcom.github.io/sfdx-scanner/en/v3.x/code-analyzer-vs-code-extension/) documentation.
```

Comment:
How about we put the link to the published docs here, rather than to the source MD file?  

https://developer.salesforce.com/docs/platform/salesforce-code-analyzer/guide/code-analyzer-vs-code-extension.html

Reply:
Agreed. Yes please update this instead of relying on the old redirects.

Reply:
Oh I didn't realize that!  Thx for letting me know